server/container/guice/rabbitmq/src/main/java/org/apache/james/modules/rabbitmq/RabbitMQModule.java

```diff
@@ -40,6 +61,40 @@
 
     @Override
     protected void configure() {
+        bind(Clock.class).toInstance(Clock.systemUTC());
+        bind(ThreadLocalRandom.class).toInstance(ThreadLocalRandom.current());
+        bind(CassandraMailQueueViewConfiguration.class).toInstance(CassandraMailQueueViewConfiguration
```

Comment:
Should be read from `rabbitmq.properties`, isn't it?

Reply:
I propose my fixup in #1776 . Check it out!

Reply:
Update: simplify loading configuration, fixup from above PR: https://github.com/linagora/james-project/pull/1776/commits/4134d4c8e5c1df6dd610d35c6a2085d04ef0ef6f